docs/glossary.md

```diff
@@ -0,0 +1,53 @@
+# Terminology
+
+### Jobs 
+
+Jobs, or Primary Jobs, are special types of tasks run in simulations.
+
+### Tasks
+
+Tasks are any thing run within simulations.
+
+### Taskflow
+
+Taskflows are a collection of tasks and jobs which are run for simulations.
+
+### Workflow
+
+Workflows are collections of steps which define some pattern for executing a process.
+
+### Projects
+
+Projects are collections of simulations.
+
+### Simulations
+
+Simulations are collections of workflows.
+
+### Clusters
+
+Clusters are any machine that you can run tasks on.
+
+### Traditional Clusters
+
+Traditional clusters are physical hardware dedicated to running simulations within the host operating system.
+
+### AWS Profile
+
+AWS (Amazon Web Services) profiles consist of a Access key and a Secret key with which a user can provision EC2 instances.
+
+### EC2 Instances
+
+EC2 Instances are clusters, or groups of clusters, running in Amazon's EC2 Cloud.
+
```

Comment:
EC2 instances usually refer to the individual virtual machines rather than a cluster.